packages/grid/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { humanize } from '@sanomalearning/slds-core/utils/string.js';
+
+export const getNameByPath = (path?: string): string => {
+  if (!path) {
+    return 'No path set';
+  } else {
+    const parts = path.split('.');
+
+    return humanize(parts[parts.length - 1]);
+  }
+};
+
+export const getValueByPath = (object: unknown, path = ''): unknown => {
+  const parts = path?.split('.');
+
+  let result = object;
+  for (const part of parts) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+    result = (result as any)[part];
```

Comment:
Can we get rid of this `any` here?

Reply:
If you know how I could do the types here, sure. But I don't know how to fix the types so I don't need `any`.